components/well-bold/well-bold-doc-fr.html

```diff
@@ -0,0 +1,79 @@
+---
+breadcrumb: [
```

Comment:
Il y a un "s" pour la propriété breadcrumbs. ref.  https://github.com/wet-boew/GCWeb/blob/master/sites/breadcrumbs/inc-breadcrumbs.html#L12
```suggestion
breadcrumbs: [
```